eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -224,12 +222,12 @@ private boolean isEvaClusteredAccession(Long accession) {
      * Note that for submitted variants the test is not this simple, as 1:1000:A:T and 1:1000:A:G can be present in the
      * same assembly and still not classify as multimap.
      */
-    private boolean isMultimap(List<? extends IClusteredVariant> clusteredVariantLoci) {
-        int assembliesCount = clusteredVariantLoci.stream()
-                                                  .map(IClusteredVariant::getAssemblyAccession)
-                                                  .collect(Collectors.toSet())
-                                                  .size();
-        return assembliesCount < clusteredVariantLoci.size();
+    private boolean isMultimap(List<? extends IClusteredVariant> clusteredVariants) {
+        return clusteredVariants.stream().anyMatch(cv -> cv.getMapWeight() != null && cv.getMapWeight() > 1);
+    }
+
+    private boolean isMultimap(IClusteredVariant clusteredVariant) {
+        return isMultimap(Collections.singletonList(clusteredVariant));
```

Comment:
I'm not sure why there are two functions: one for a list and one for a single item. The only used one I can see is for the single item. I would suggest removing the one taking a list.  
```suggestion
    private boolean isMultimap(IClusteredVariant clusteredVariant) {
        return clusteredVariant.getMapWeight() != null && clusteredVariant.getMapWeight() > 1;
```

Reply:
The method that receives the list is used in the `merge` method. I could keep only one if you think this is creating confusion but it would be better to keep the one that takes the list, otherwise we would need to include loops when calling it. My preference is to keep both but let me know if you still think this makes the code less clear.

Reply:
Yes you are right. I missed that. The code makes sense as it is.